src/encoded/docs/schema-changes.md

```diff
@@ -154,44 +154,115 @@ This new object is an array of example objects that can successfully POST agains
 
-**The schema version is updated, requiring an upgrade of the schema, when the objects stored in the database will not validate and POST to the updated schema.**
+**Follow the steps as outlined below**
 
+1. In the **schemas** directory, edit the existing properties in the corresponding JSON file named after the object.
 
-### No affect on schema version
+2. In the **types** directory, make appropriate updates to object class by adding *embedding*, *reverse links*, and *calculated properties* as necessary.
 
-1. In the **schemas** directory, make edits  to properties in the JSON file named after the object.
+3. Update the inserts within the **data/inserts** directory.
 
-2. In the **types** directory, make appropriate updates to object class by adding *embedding*, *reverse links*, and *calculated properties* as necessary.
+4. Add test in the **tests** directory to make sure your schema change functions as expected. For example, if we included a minor change in treatment object such that *μg/kg* could be specified as treatment units, the following test should allow one to test whether the update has been successfully implemented or not:
+
+        def test_treatment_patch_amount_units(testapp, treatment):
+            testapp.patch_json(
+            treatment['@id'],
+            {
+                'treatment_type': 'injection',
+                'amount': 20,
+                'amount_units': 'μg/kg'
+            },
+            status=200, 
+            ## Status 200 means the object was a successfully patched and the schema update works as expected.
+        )
 
-3. Update sample data, **data/inserts** directory, to test the changes made to the schema in **tests** directory. 
+5. Document the changes to the corresponding log file within the **schemas/changelogs** directory. For example, a minor change in the treatment object after version 11 that allowed one to use *μg/kg* as treatment units is shown below:
 
+### Minor changes since schema version 11
+
+* *μg/kg* can now be specified as amount units.
 
 ### Update schema version
+* Schema version has to be updated (bumped up by 1) if the change that is being introduced will lead to a potential invalidation of existing objects in the database. 
+
+* Examples include:
+    1) Changing the name of a property in the existing schema.
+    2) Removing a property from the existing schema.
+    3) A property that previously allowed free text is now changed to a possible list of allowed enums.
+    4) If an existing enum is removed from a property.
+    5) If one object is migrated into another object.
+    6) Addition of multiple new schema properties leading to substantial changes to an existing schema.
```

Comment:
that would indeed bump the schema number, but not because objects are going to be invalidated, hence it should appear separately from the rest